modules/javafx.web/src/test/java/test/javafx/scene/web/HistoryTest.java

```diff
@@ -95,21 +95,6 @@ public void onChanged(ListChangeListener.Change<? extends WebHistory.Entry> c) {
 
         ensureValueChanged(entriesChanged, "entries not changed after load");
 
-        //
-        // check the title update
-        //
```

Comment:
Since this uses a listener to read the title property, why did this check need to be removed? We might want to file a P4 test bug to look into adding it back in. To that end, do you think it is better to comment this block of code out rather than delete it?

Reply:
I have reverted this change and commented the test and created [JDK-8269912](https://bugs.openjdk.java.net/browse/JDK-8269912) to investigate test.